### PR TITLE
Expose kyma version on runtimes endpoint

### PR DESCRIPTION
**Pull request by ebensom**
**Description**

Changes proposed in this pull request:

- Expose kyma version on KEB `/runtimes` endpoint from the operation `RuntimeVersion` data. Only applicable when all operations are queried.
- Adjust KCP CLI to optionally display kyva version in default table format.

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/runtime/handler.go [hunk 1]
```diff
@@ -172,6 +175,9 @@ func (h *Handler) setRuntimeAllOperations(instance internal.Instance, dto *pkg.R
 		if len(provOprs) > 1 {
 			h.converter.ApplyUnsuspensionOperations(dto, provOprs[:len(provOprs)-1])
 		}
+		// Set kyma version from the last provisioning operation
+		kymaVersion = lastProvOp.RuntimeVersion.Version
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/runtime/handler.go:179]: don't upgrade operations also kyma runtime version? Would it make sense to take the last provisioning or upgrade operation to determine the current kyma version of the SKR?

**Review by wozniakjan — APPROVED** [review]
/lgtm
/hold

feel free to unhold if you think my comment doesn't apply

File: components/kyma-environment-broker/internal/runtime/handler.go (hunk 1 above)
Comment (ebensom) [reply components/kyma-environment-broker/internal/runtime/handler.go:179]: Yes, it's exactly what I'm doing a few lines below this change. If there are upgrade kyma operations and the last upgrade operation happened after the last provision/unsuspension operation, take it's kyma version data instead.

Comment (ebensom) [reply components/kyma-environment-broker/internal/runtime/handler.go:179]: Or did I miss something?

Comment (wozniakjan) [reply components/kyma-environment-broker/internal/runtime/handler.go:179]: oh, my bad, I missed that. Ignore me :)

**Pull request merged**